lxd/db/backups.go

```diff
@@ -282,6 +282,49 @@ func (c *Cluster) GetExpiredInstanceBackups() ([]InstanceBackup, error) {
 	return result, nil
 }
 
+// GetExpiredStorageVolumeBackups returns a list of expired storage volume backups.
+func (c *Cluster) GetExpiredStorageVolumeBackups() ([]StoragePoolVolumeBackup, error) {
+	var result []StoragePoolVolumeBackup
+	var name string
+	var expiryDate string
+	var storageVolumeID int64
+
+	q := `SELECT storage_volumes_backups.name, storage_volumes_backups.expiry_date, storage_volumes_backups.storage_volume_id FROM storage_volumes_backups`
+	outfmt := []any{name, expiryDate, storageVolumeID}
+	dbResults, err := queryScan(c, q, nil, outfmt)
```

Comment:
Please can you use `query.Scan` instead of `queryScan` as it avoids the need to use the unwieldy `outFmt` and type assertions.